src/aind_metadata_service/labtracks/client.py

```diff
@@ -324,6 +330,31 @@ def _map_to_background_strain(
         else:
             return None
 
+    @staticmethod
+    def _map_housing(
+        room_id: Optional[str], cage_id: Optional[str]
+    ) -> Optional[Housing]:
```

Comment:
All the fields in the Housing model are optional. It might be simpler to just return `Housing(room_id=room_id, cage_id=cage_id)`

Reply:
Once we have a way to get the light cycle info in, I would like to make it required. It's only been optional because it hasn't been possible. In case that matters.

Reply:
Cool. We can leave the code as it is then.